packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
+    auth_mode: OAUTH2
+    authorization_url: https://acuityscheduling.com/oauth2/authorize
+    token_url: https://acuityscheduling.com/oauth2/token
+    connection_configuration:
+        - userId
+        - apiKey
```

Comment:
You can remove the post connection script and webhook routing script since it doesn’t have either. I realize all the parameters can be confusing - so sorry about that!

Reply:
haha no worries! I'll take it out